playwright/tests/rex-test/rex.behaviorspec.ts

```diff
@@ -103,21 +103,55 @@ test('multiple highlight', async ({ page, isMobile }) => {
   await rexUserSignup(page)
   await expect(page).toHaveURL('/books/introduction-anthropology/pages/7-introduction')
 
-  // WHEN: Highlight any random paragraph
+  // WHEN: Highlight 2 random paragraphs
   const paracount = BookPage.paracount()
   const randomparanumber = randomNum(await paracount)
   await BookPage.highlightText('green', randomparanumber)
 
-  // THEN: Text is highlighted
-  let highlightcount = await BookPage.highlightCount()
-  expect(highlightcount).toBe(1)
-
-  // AND: Highlight another random paragraph
-  await BookPage.scrolltotop()
   const randomparanumber2 = randomNum(await paracount, randomparanumber)
   await BookPage.highlightText('yellow', randomparanumber2)
 
   // THEN: Text is highlighted
-  highlightcount = await BookPage.highlightCount()
+  let highlightcount = await BookPage.highlightCount()
   expect(highlightcount).toBe(2)
+
+  // AND: Navigate to next page
+  await BookPage.clickNext()
+
+  // WHEN: Highlight 2 random paragraphs
+  const paracount3 = BookPage.paracount()
+  const randomparanumber3 = randomNum(await paracount3)
+  await BookPage.highlightText('green', randomparanumber3)
+
+  const randomparanumber4 = randomNum(await paracount3, randomparanumber3)
+  await BookPage.highlightText('yellow', randomparanumber4)
+
+
+  // WHEN: Open MH modal
+  await BookPage.openMHmodal()
+
+  const Modal = new MHModal(page)
+  await expect(Modal.MHModal).toBeVisible()
+
+  
+  const Edithighlight = new EditHighlights(page)
+  sleep(1)
+  let MHhighlightcount = await Edithighlight.highlightCount()
+  expect(MHhighlightcount).toBe(4)
+
+
+  // let x = await Edithighlight.highlightlist()
+  // console.log(x[1])
+  // const highlight = x[1].toString().replace(/\D/g, "")
+  // console.log(highlight)
+
+  await Edithighlight.clickContextMenu(1)
+  sleep(5)
```

Comment:
@gregfitch Please review lines 137 - 148. Is this the right way to open the context menu of a highlight?
So the number we are passing as a parameter in 148 is what determines the highlight in the modal.

Reply:
I may be reading the structure incorrectly (MHmodal.ts), but it looks like you're doing something different than I would expect. Your comment lists the class EditHighlights as a highlight, but it's extending the whole modal; why can't this live in one of the other classes?

Reply:
As to the code, yes, this should get the nth element as long as you adjust `n` to be one less (1st element is n=0); clickContextMenu(1) should open the second highlight toggle menu.

Reply:
@gregfitch can you please review again. I removed the toolbar class